lib/blobs/usbboot/config.txt

```diff
@@ -0,0 +1,6 @@
+kernel=kernel7.img
```

Comment:
This line ought to be removed, so that we can use the same config.txt for both CM1 and CM3.

Reply:
Will it still work if I remove this?

Reply:
On the CM3 it defaults to loading `kernel7.img`, and on the CM1 it defaults to loading `kernel.img`, which is exactly what we want. If we leave this line in place, it'll try to use `kernel7.img` for both CM1 and CM3, which will fail when we try to boot an ARMv7 `kernel7.img` on the ARMv6 of the CM1.

Reply:
Awesome, done!